metadata-ingestion/docs/sources/powerbi/powerbi_recipe.yml

```diff
@@ -23,10 +23,18 @@ source:
     extract_endorsements_to_tags: false
     # dataset_type_mapping is fixed mapping of Power BI datasources type to equivalent Datahub "data platform" dataset
     dataset_type_mapping:
-        PostgreSql: postgres
-        Oracle: oracle
-        Sql: mssql
-        GoogleBigQuery: bigquery
+        PostgreSql:
```

Comment:
The whole purpose is to make the user do LESS. Why would we include this in our sample recipe? Please reove the entire field from our starter recipe

Reply:
This to give example for how to set the platform_instance with recipe, I will remove it in next PR when we make this field deprecated

Reply:
It is for reference for how to set platform_instance. We can remove this field once we deprecated it in next PR.